.github/workflows/pull_request_build.yml

```diff
@@ -33,10 +32,49 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-sonar-
 
-    - name: Set up JDK 1.8
+    - name: Setup JDK 1.8
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
 
+    - name: Build P2
+      run: bash ./modules/p2/prepareDependenciesForP2.sh
+
     - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      run: mvn -B clean install -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+    - name: Run SCA Analysis
+      run: mvn verify -DskipTests pmd:pmd checkstyle:checkstyle-aggregate spotbugs:check
+
+#    - name: Run SonarCloud Analysis
```

Comment:
I'd prefer to not commit any out-commented code. Also see e.g. https://kentcdodds.com/blog/please-dont-commit-commented-out-code.